Sources/Networking/Operations/HealthReportOperation.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  HealthReportOperation.swift
+//
+//  Created by Pol Piella on 4/8/25.
```

Comment:
one of us one of us 
![one of us](https://media4.giphy.com/media/v1.Y2lkPTc5MGI3NjExOHl4ZXkxN3F6b2Jhdmt2aHp5bXg2NXJmbzUzaHpzbXowd2EzdDk2ZyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/Ae7SI3LoPYj8Q/giphy.gif)